lib/pantry/handlers/transaction_intent_request_handler.rb

```diff
@@ -50,19 +46,29 @@ def addItems
           item.save
           item
         end
+        Helpers::HandlerHelper.create_response(
+          message: 'Items successfully added',
+          card: {
+            type: 'Simple',
+            title: 'Added Items',
+            content: Helpers::HandlerHelper.prepare_items_for_card_without_date(categorized_items)
+          },
+          end_session: true
+        )
       end
 
       # TODO, this will be implemented later
-      def removeItems(items)
+      def remove_items
 
       end
 
-      def getItemsFromSlots
+      def get_items_from_slots
         slots.reduce(Hash.new) do |hash, (k,v)|
           if v['value'].present?
             quantity = v['value'].scan( /\d+/ ).first.to_i
             quantity.present? ? quantity : 1
-            hash.merge( {v['value'] => quantity} )
+            value = v['value'].sub(/\d+/, '').squeeze(' ').strip
+            hash.merge( {value => quantity} )
```

Comment:
Any time you're calling a method in Ruby and the arguments are a hash, you can skip the `{ }`:
```ruby
hash.merge(value => quantity)
```